### PR TITLE
New branch for CP review

### PR DESCRIPTION
**Pull request by mrcbq**
In this branch I did:
Copy my code from the "To Do list" project and commit it as one big chunk of code in a feature branch.
Open a pull request.
Ask you partner for a review in [GitHub](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review).
Share the pull request link with your partner.

### CODE REVIEW

File: src/methods.js
Comment (hajnaloltyan) [inline src/methods.js:1]: I think similar to the filterTasks function, you can use arrow functions for all of them to look unified and use modern JS. So kindly please change all to arrow functions.

File: dist/index.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  resize: none;
+}
+
+li > input {
+  height: 1.5rem;
+}
+
+.input-btn-ccw {
+  width: 1.5rem;
+  height: 1.5rem;
```
Comment (hajnaloltyan) [inline dist/index.css]: I can see you are using the width and height in your css multiple places with the same 1.5 rem, for this to comply with DRY I would kindly suggest to create a separare class if possible.

**Review by hajnaloltyan — CHANGES_REQUESTED** [review]
Hi @mrcbq  wave , I am @hajnaloltyan, your Code Reviewer.

Your project is amazing, functions have descriptive names and everything does what it supposed to. I found only 2 minor issues that I would suggest you take into account and kindly fix.

For details, please see the comments below....

File: dist/index.css (hunk 1 above)
Comment (mrcbq) [reply dist/index.css]: Thanks for your comments.
It's done.

File: src/methods.js
Comment (mrcbq) [reply src/methods.js:1]: Thanks for your comments.
It's done.

**Review by hajnaloltyan — APPROVED** [review]
Hi @mrcbq , Great job fixing it. Congratulations, your project is finised, now it's time to merge it! 👍🏼

**Pull request merged**